dali/base/dautils.cpp

```diff
@@ -3245,9 +3245,25 @@ class CLocalOrDistributedFile: implements ILocalOrDistributedFile, public CInter
                 if (dfile.get())
                     return true;
             }
+
+            StringBuffer dir;
+            StringBuffer cluster;
```

Comment:
cluster should be inside the #ifdef - otherwise it will be an unreferenced object

Reply:
right, I'll move it.
gcc doesn't complain - does clang? Is it because compiler not smart enough to realize because object has ctor etc. ?